src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -305,7 +305,6 @@ function createOpenTagMarkup(
   if (isRootElement) {
     ret += ' ' + DOMMarkupOperations.createMarkupForRoot();
   }
-  ret += ' ' + DOMMarkupOperations.createMarkupForID('');
```

Comment:
A child of the container. The React root is the first child of the container. It also has children. You're not supported to render into those.